plugins/janus_streaming.c

```diff
@@ -148,6 +149,37 @@ rtspiface = network interface IP address or device name to listen on when receiv
  * be rejected otherwise.
  *
  * Actual API docs: TBD.
+ *
+ * Once a mountpoint has been created, you can still edit some (but not all)
+ * of its properties using the \c edit request. This allows you to modify
+ * the mountpoint description, secret, pin and whether it's private or not: you
+ * won't be able to modify other more static properties, like the mountpoint ID,
+ * the sampling rate, the extensions-related stuff and so on. If you're
+ * interested in changing the ACL, instead, check the \c allowed message.
+ * An \c edit request has to be formatted as follows:
+ *
+\verbatim
+{
+	"request" : "edit",
+	"id" : <unique numeric ID of the mountpoint to edit>,
+	"secret" : "<room secret, mandatory if configured>",
+	"new_description" : "<new pretty name of the mountpoint, optional>",
+	"new_secret" : "<new password required to edit/destroy the mountpoint, optional>",
+	"new_pin" : "<new password required to join the mountpoint, optional>",
+	"new_is_private" : <true|false, whether the mountpoint should appear in a list request>,
+	"permanent" : <true|false, whether the mountpoint should be also removed from the config file, default false>
+}
+\endverbatim
+ *
+ * A successful destruction procedure will result in an \c edited response:
+ *
+\verbatim
+{
+	"mountpoint" : "edited",
+	"id" : <unique numeric ID>,
+	"permanent" : <true|false>
+}
+\endverbatim
```

Comment:
It's a bit weird to see documentation for a single method, considering all the others are missing... (`Actual API docs: TBD.`) I guess you can leave this out until the whole API is documented.

Reply:
Totally, but my thoughts we're we might as well guess this started!

Reply:
I'd rather see this happen in a different PR that adds a documentation for everything: having a single thing in would look weird in the generated docs.